pkg/adaptor/hypervisor/libvirt/libvirt.go

```diff
@@ -135,8 +138,7 @@ func CreateInstance(c context.Context, libvirtClient *libvirtClient, v *vmConfig
 
 	exists, err := checkInstanceExistsByName(v.name, libvirtClient)
 	if err != nil {
-		logger.Printf("Error in checking instance ")
-		return nil, err
+		return nil, fmt.Errorf("Error in checking instance: %s", err)
```

Comment:
How about using a consistent mechanism across the code. 
Either we use the following: 
```
logger.Printf("Error in checking instance: %s", err)
return nil, err
```
or
```
return nil, fmt.Errorf("Error in checking instance: %s", err)
```
Currently the code is using a mixed approach. Since anyway we are changing, it might be best to use a consistent format.